components/calendar/index.css

```diff
@@ -9,31 +9,100 @@ the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTA
 OF ANY KIND, either express or implied. See the License for the specific language
 governing permissions and limitations under the License.
 */
+.spectrum--medium {
+  --spectrum-calendar-day-width: 32px;
+  --spectrum-calendar-day-height: 32px;
+  --spectrum-calendar-button-gap: 3px;
+  --spectrum-calendar-title-text-size: 18px;
+  --spectrum-calendar-day-title-text-size: 11px;
+  --spectrum-calendar-day-text-size: 14px;
+  --spectrum-calendar-day-text-size-han: 11px;
+}
+.spectrum--large {
+  --spectrum-calendar-day-width: 40px;
+  --spectrum-calendar-day-height: 40px;
+  --spectrum-calendar-button-gap: 4px;
+  --spectrum-calendar-title-text-size: 22px;
+  --spectrum-calendar-day-title-text-size: 13px;
+  --spectrum-calendar-day-text-size: 17px;
+  --spectrum-calendar-day-text-size-han: 13px;
+}
+
+.spectrum--light {
+  --spectrum-calendar-day-background-color-selected: rgba(2,101,220,0.1);
```

Comment:
Well that's super annoying. I feel like they would want those to align rather than be different for dark mode but that's not my call to make. Hopefully design changes that when they look into this one!